imap/jmap_mail.c

```diff
@@ -14562,33 +14572,29 @@ static int jmap_email_copy(jmap_req_t *req)
     srcinbox = mboxname_user_mbox(copy.from_account_id, NULL);
     dstinbox = mboxname_user_mbox(req->accountid, NULL);
 
+    int r = conversations_open_user(copy.from_account_id, 0, &from_cstate);
+    if (r) {
+        syslog(LOG_ERR,
+               "jmap_email_copy: can't open source conversations: %s",
```

Comment:
Use xsyslog here and maybe replace this with something like "can't open conversations: fromAccountId=<%s> error=<%s>"?